style.css

```diff
@@ -35,64 +35,135 @@ body {
   z-index: 1;
 }
 
-/* #menu-option {
-  z-index: 10;
-} */
-
-/* .nav-bar_desktop {
-  display: none;
-  position: absolute;
-  height: 100%;
-  width: 100%;
-  flex-direction: column;
-  background-color: #fff;
-  z-index: 10;
-} */
-
 .hidden {
   display: none;
 }
 
-/* .nav-bar_desktop ul {
+.nav-bar img {
+  height: 20px;
+}
+
+/* POPUP-MOBILE START HERE */
+
+ul {
   list-style: none;
-  color:#3a4a42;
-  padding-left: 24px;
-  padding-top: 24px;
-} */
+}
+
+.popup {
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  backdrop-filter: blur(8px);
+  transition: top 3s ease-in-out;
+}
+
+.popup-content {
+  width: 90%;
+  margin: 30px auto;
+  height: auto;
+  background-color: #fff;
+  justify-content: center;
+  top: 60px;
+  padding: 5px 20px;
+  border-radius: 8px;
+  text-align: center;
+}
 
-/* .nav-bar_desktop li {
-  margin-bottom: 25px;
-} */
+.close-btn {
+  float: right;
+  font-size: 20px;
+  color: #c1c7d0;
+  cursor: pointer;
+}
+
+.pop-img {
+  width: 100%;
+  height: 100%;
+}
 
-/* .close {
+.pop-heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   margin-top: 20px;
-  align-self: flex-end;
-  margin-right: 20px;
 }
-.x-mark {
-  height: 15px;
-  width: 15px;
-  color:#3a4a42;
-  z-index: 10;
-} */
-
-/* .nav-bar_desktop a {
-  text-decoration: none;
-  color:#3a4a42;
-  font-family: 'Inter';
-  font-weight: 600;
+
+.pop-heading h3 {
+  font-family: var(--font-family);
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
   font-size: 32px;
   line-height: 44px;
+  display: block;
+  text-align: left;
+}
+
+.pop-button {
+  display: none;
+}
+
+.pop-button-mob {
   display: flex;
-  align-items: center;
-  padding-left: 16px;
-  z-index: 10;
-} */
+}
 
-.nav-bar img {
-  height: 20px;
+.button-info-mob {
+  padding: 15px;
+  font-size: 15px;
+  font-weight: 600;
+  border-radius: 8px;
+  border: none;
+  color: #fff;
+  background-color: #36b37e;
+  margin-right: 20px;
+}
+
+.button-info img {
+  margin-top: 10px;
+}
+
+.pop-skills {
+  display: none;
+}
+
+.pop-skills-mob {
+  margin-top: 20px;
+}
+
+.pop-skills-mob ul {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+}
+
+.pop-skills-mob li {
+  padding: 8px 12px;
+  background-color: #ebecf0;
+  border-radius: 4px;
+  color: #344563;
+  font-size: 12px;
+  font-weight: 600;
+  font-family: var(--font-family);
+}
+
+.pop-details {
+  text-align: left;
+  line-height: 30px;
+  font-family: var(--font-family);
+  font-weight: 400;
+  font-size: 16px;
+  color: #344563;
+  margin-right: 20px;
+  margin-top: 30px;
```

Comment:
- [ ] Kindly add more space below the text in the image and also add the writings highlighted in red in the image, so as to make your site look more like the given design

### Your design
![Screenshot 2023-04-22 at 23 37 58 (2)](https://user-images.githubusercontent.com/64774936/233810838-189d8437-5ad1-46d8-a0a3-6fd70c70f17f.png)

### Given design
![Screenshot 2023-04-22 at 23 38 06 (2)](https://user-images.githubusercontent.com/64774936/233810847-65f19442-7973-48ba-9d04-ee95bd3b80e7.png)